### PR TITLE
[PM-8125] Move Trash to the Vault settings page

### PR DESCRIPTION
**Pull request by gbubemismith**
## 🎟️ Tracking
https://bitwarden.atlassian.net/browse/PM-8125

## 📔 Objective
Move Trash component to the Trash page.

## 📸 Screenshots

https://github.com/user-attachments/assets/c29f484c-a46b-4564-93e5-e0f8868f309b

| Light      | Dark |
| ----------- | ----------- |
|image|image|

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/browser/src/vault/popup/settings/trash-list-items-container/trash-list-items-container.component.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<bit-section *ngIf="ciphers?.length">
+  <bit-section-header>
+    <h2 class="tw-font-bold" bitTypography="h5">
```
Comment (danielleflinn) [inline apps/browser/src/vault/popup/settings/trash-list-items-container/trash-list-items-container.component.html:3]: This should be `bitTypography="h6"` and I think the `tw-font-bold` class is not needed. The font-weight should be applied from the bitTypography element already.

The styling of the section header should look the same as the section header on the cipher Edit/View pages.

**Review by danielleflinn — CHANGES_REQUESTED** [review]
One design bug I caught; otherwise looks good. Thanks!

**Review by danielleflinn — APPROVED** [review]

File: apps/browser/src/vault/popup/settings/trash-list-items-container/trash-list-items-container.component.html (hunk 1 above)
Comment (danielleflinn) [reply apps/browser/src/vault/popup/settings/trash-list-items-container/trash-list-items-container.component.html]: @gbubemismith can you update the PR screen recording or drop a screenshot here of the update? 

I'm OOO the next few days so I went ahead and approved this though so you aren't blocked while I'm out

Comment (gbubemismith) [reply apps/browser/src/vault/popup/settings/trash-list-items-container/trash-list-items-container.component.html]: @danielleflinn, the PR has been updated with the latest screen recording. Thank you.

**Review by Jingo88 — APPROVED** [review]

**Pull request merged**